### PR TITLE
fix(ux/portal): conditions ux tweaks

### LINKED ISSUE #6367: Minor policy conditions UX
- [ ] `Restrict access based on...` is a bit ambiguous as to whether we require the conditions to be true or false to allow access.
- [ ] Use map-pin icon for location to free up Globe for other things

### PR DESCRIPTION
**Pull request by jamilbk**
fixes #6367 

Screenshot 2024-08-22 at 1 58 20 PM

### CODE REVIEW

File: elixir/apps/web/lib/web/live/policies/components.ex [hunk 1]
```diff
@@ -235,7 +235,12 @@ defmodule Web.Policies.Components do
     ~H"""
     <fieldset class="flex flex-col gap-2 mt-4">
       <div class="flex items-center justify-between">
-        <legend class="text-xl mb-2 text-neutral-900">Conditions</legend>
+        <div>
+          <legend class="text-xl mb-2 text-neutral-900">Conditions</legend>
+          <p class="my-2 text-sm text-neutral-500">
+            When specified, all conditions below must be met for this policy to be applied.
```
Comment (bmanifold) [inline elixir/apps/web/lib/web/live/policies/components.ex]: English is so ambiguous at times 😄 
Do you think someone could interpret this as "if any are specified then all must be met..."?

**Review by bmanifold — APPROVED** [review]
LGTM, just one non-blocking question

File: elixir/apps/web/lib/web/live/policies/components.ex (hunk 1 above)
Comment (jamilbk) [reply elixir/apps/web/lib/web/live/policies/components.ex]: Yeah, but that's the intention. Maybe `If specified, ...`?

Comment (bmanifold) [reply elixir/apps/web/lib/web/live/policies/components.ex]: What do you think about "All specified conditions below..." ?
Honestly, I don't have strong feelings, just a quick gut check is all.

Comment (jamilbk) [inline elixir/apps/web/lib/web/live/policies/components.ex]:
```suggestion
            All conditions specified below must be met for this policy to be applied.
```

**Pull request merged**